tests/ci/codebuild/linux-x86/centos-7_gcc-4x-32-bits.yml

```diff
@@ -10,7 +10,6 @@ phases:
       - export CC=gcc
       - export CXX=g++
       - export AWSLC_32BIT=1
```

Comment:
Does it make sense to make this yaml file generic and name it run_32bit_posix_tests.yml?

Reply:
Addressed.